models/playlist.ts

```diff
@@ -17,4 +17,41 @@ module.exports.savePlaylist = function (name, songs, user_id, type, startdatetim
             return callback(err, result);
         }
     })
+};
+module.exports.saveOrUpdatePlaylist = function (name, songs, user_id, type, startdatetime, enddatetime, callback) {
+    var playlistFound = false;
+    pool.query("SELECT user_id ,name FROM playlists", function (err, result, fields) {
+        if (err) {
+            Logger.log('error', err);
+            callback(err, '');
+        }
+        else {
+            for (var i = 0; i < result.rows.length; i++) {
+                if (result.rows[i].user_id == user_id && result.rows[i].name == name) {
+                    var sql = 'DELETE FROM playlists WHERE user_id = $1 AND name = $2';
```

Comment:
Als je hem echter delete dan heb je uberhaupt geen reden om verder daarvoor of daarna te checken of die bestaat... waarom is dat nodig?